EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/36397
+itcafe.hu#?#div[class$="responsive-300px-600px"]:has( > h4:contains(/H*s/))
```

Comment:
itcafe.hu##div[class$="responsive-300px-600px"]

Those two values are normal ad values, so it should be safe to use no ext-CSS.
FYI if a non-English site requires usage of ext-CSS rules, please put them into foreign.txt too.